llvm/lib/ProfileData/SampleProfReader.cpp

```diff
@@ -791,6 +798,19 @@ bool SampleProfileReaderExtBinaryBase::useFuncOffsetList() const {
   return false;
 }
 
+std::error_code
+SampleProfileReaderExtBinaryBase::read(const DenseSet<StringRef> &FuncsToUse,
```

Comment:
We also need to make sure if we profile is loaded, we don't load again into the same map